vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -22,11 +25,16 @@
 //     'New Project' --> 'Online Templates'
 
 
-// Open namespaces using 'open'
+// Open namespaces using the 'open' directive.
+//
+// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/import-declarations-the-open-keyword
 open System
 
 
-/// First some basics on integers and numbers
+/// Modules are the primary way to organize functions and values in F#. This module contains some
```

Comment:
Should we explain the difference between // comments and triple-slash comments? It looks like a mix has been used in this document which could be confusing as to what the 'defacto' comment style is, and when/where each type should be used.